.github/ISSUE_TEMPLATE/usage_question.yaml

```diff
@@ -0,0 +1,66 @@
+name: Usage Question
+description: Ask a question
+labels: ["Type: usage"]
```

Comment:
I am not in love with that label format but as we can change this pretty easily later I don't think its a blocker (as we want to add this asap due to the locked down JIRA)